test/c/custom_flow.sail

```diff
@@ -3,6 +3,7 @@ val "print_endline" : string -> unit
 
 val operator <= = {
   coq: "Z.leb",
+  lean: "Int.lteq",
```

Comment:
You can just use `LE.le` from the standard library, you don't need to define a the new `lteq` function.